tests/test_dse.py

```diff
@@ -394,7 +396,7 @@ def test_full_shape_with_subdims(self):
         f = Function(name='f', grid=grid)
         f.data_with_halo[:] = 1.
         u = TimeFunction(name='u', grid=grid, space_order=3)
-        u.data_with_halo[:] = 0.
+        u.data_with_halo[:] = 0.5
```

Comment:
?

Reply:
to minimize amount of computation on zeros